pkg/middlewares/google_iam_binding_transformer.go

```diff
@@ -39,7 +39,11 @@ func (m *GoogleIAMBindingTransformer) Execute(_, resourcesFromState *[]*resource
 
 		resName := *stateRes.Attrs.GetString(resField)
 		roleName := *stateRes.Attrs.GetString("role")
-		members, _ := stateRes.Attrs.Get("members")
+		members, exist := stateRes.Attrs.Get("members")
```

Comment:
Can the role be nil?

Reply:
good question I don't think so

Reply:
I have really no idea, I've not worked on this one. I've not found the root cause of why do we have an empty members key here in the state but I was supposing that it was either a `default [] vs nil` issue or a state refresh problem.

I would say that without a role this resource does not make sense so I don't think that it can be empty at some point, @moadibfr any idea ?

Reply:
okm from tf doc members AND role are required so maybe it would be safe to test both ?

Reply:
I asked the reporter for details, maybe it will help let's see if we can get the full resource definition in the state

Reply:
Anyway, I'm wondering if we should introduce a mechanism to recover from theses panics in middleware pass and send alerts WDYT ?

Reply:
I thought we already had existing sentry functionality wrapped around the whole invocation for exactly this? (if the user opts into crash reports)

Are you talking about a different alerting mechanism?

Reply:
Ho yeah sorry for the poor context @craigfurman 🙏🏻 we have another internal mechanism for alerting  the user that something went wrong during the scan without interrupting it. This mechanism also allows us to ignore a resource or a whole complete resource type from the computation too (to avoid noise for resource types that encountered some errors). But good point about sentry, if we recover here we wont see theses issues in sentry, maybe it can be nice to manually send them in sentry for theses case event if we catch the panic, because it's still something unexpected.